code/modules/medical/diseases/plasmatoid.dm

```diff
@@ -7,3 +7,54 @@
 	recureprob = 15
 	associated_reagent = "liquid plasma"
 	affected_species = list("Monkey", "Human")
+
+/datum/ailment/disease/plasmatoid/stage_act(mob/living/affected_mob, datum/ailment_data/D)
+	if (..())
+		return
+	switch(D.stage)
+		if(2)
+			if(prob(1))
```

Comment:
A few prob(1)s doesn't really feel like much in the way of a warning, might be different in practice though

Reply:
Its prob() all the way down....

I restructured so you are more likely to get some symptom but it is possible to make it all the way without a single symptom.